sage-ios/SAGE/SAGE/User.swift

```diff
@@ -27,6 +27,7 @@ class User: NSObject, NSCoding {
     enum DefaultValues: Int {
         case DefaultID = -1
         case DefaultHours = -2
+        case DefaultDirectorID = -3
```

Comment:
thought it would be easier to check against variable values -- should i change it back to -1?